packages/shared/src/components/search/SearchPanel/SearchPanelSourceSuggestions.tsx

```diff
@@ -71,7 +71,7 @@ export const SearchPanelSourceSuggestions = ({
   });
 
   const onSuggestionClick = (suggestion: SearchSuggestion) => {
-    const source = suggestion.id || suggestion.subtitle.toLowerCase();
+    const source = suggestion.subtitle.toLowerCase() || suggestion.id;
```

Comment:
I think `suggesstion.id` will never evaluate now because if `suggestion.subtitle` was undefined it would just break when calling `toLowerCase`, so maybe just?
```suggestion
    const source = suggestion.subtitle?.toLowerCase() || suggestion.id;
```